### PR TITLE
Fix build issues in frida cmplog for aarch64

### PR DESCRIPTION
**Pull request by mineo333**
This fixes a few aarch64-related regressions introduced by https://github.com/AFLplusplus/LibAFL/pull/1713

### CODE REVIEW

File: libafl_frida/src/cmplog_rt.rs [hunk 1]
```diff
@@ -384,23 +392,23 @@ impl CmpLogRuntime {
...
-    #[cfg(target_arch = "aaarch64")]
+    #[cfg(target_arch = "aarch64")]
     pub fn ops_handle_tbz_masking(&self) -> &[u8] {
         self.ops_handle_tbz_masking.as_ref().unwrap()
     }
 
     /// Get the blob which handles the tbnz opcode masking
     #[inline]
     #[must_use]
-    #[cfg(target_arch = "aaarch64")]
```
Comment (domenukk) [inline libafl_frida/src/cmplog_rt.rs:403]: loool nice catch!

Comment (mineo333) [reply libafl_frida/src/cmplog_rt.rs:403]: I was so confused lmao

**Comment by domenukk** [comment]
```rust
 use dynasmrt::dynasm;
 
 #[cfg(target_arch = "aarch64")]
-use dynasmrt::{DynasmLabelApi, DynasmApi};
+use dynasmrt::{DynasmApi, DynasmLabelApi};
 
 use libafl::{
     inputs::{HasTargetBytes, Input},
Diff in /home/runner/work/LibAFL/LibAFL/libafl_frida/src/cmplog_rt.rs at line 564:
 
         writer.put_bytes(&self.restore_registers.clone().unwrap());
     }
-
 
     /// Emit the instrumentation code which is responsible for operands value extraction and cmplog map population
     #[cfg(all(feature = "cmplog", target_arch = "aarch64"))]
```
Seems this needs a cargo fmt

**Comment by domenukk** [comment]
Thanks! :)

**Pull request merged**